assets/javascripts/discourse/components/subscribe-ca-province-select.js

```diff
@@ -0,0 +1,36 @@
+import ComboBoxComponent from "select-kit/components/combo-box";
+import { computed } from "@ember/object";
+import I18n from "I18n";
+
+export default ComboBoxComponent.extend({
+  pluginApiIdentifiers: ["subscribe-ca-province-select"],
+  classNames: ["subscribe-address-state-select"],
+  nameProperty: "name",
+  valueProperty: "value",
+
+  selectKitOptions: {
+    filterable: true,
+    allowAny: false,
+    translatedNone: I18n.t("hosted_site.invoice_details.province_select"),
```

Comment:
We should use a `subscriptions` string here.

Reply:
Updated with the previous item.